app/models/feature_flags.rb

```diff
@@ -0,0 +1,42 @@
+class FeatureFlags
```

Comment:
Instead of having a `FeatureFlags` and `Feature` class I would be inclined to just have a `Feature` class following the ActiveRecord pattern, so: `Feature.all` `Feature.enabled?(:my_feature)` etc.